src/rapidata/rapidata_client/filter/age_filter.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any
+from rapidata.rapidata_client.filter.base_filter import Filter
+from rapidata.api_client.models.age_user_filter_model import AgeUserFilterModel
+from rapidata.api_client.models.age_group import AgeGroup
+
+
+class AgeFilter(Filter):
+
+    def __init__(self, age_groups: list[str]):
```

Comment:
why are we not directly providing a list of age groups here?

Reply:
Age group is part of the generated api which I thought we generally did not want to expose. Not sure, maybe it makes sense to expose it?